src/engine/sys/sdl_glimp.cpp

```diff
@@ -1750,26 +1750,46 @@ static rserr_t GLimp_SetMode( const int mode, const bool fullscreen, const bool
 	return rserr_t::RSERR_UNKNOWN;
 }
 
+static void GLimp_SetAppMetadataProperty( const char* name, const char* value )
+{
+	if ( !SDL_SetAppMetadataProperty( name, value ) )
+	{
+		logger.Warn( "SDL_SetAppMetadataProperty( %s, %s ) failed: %s", name, value, SDL_GetError() );
+	}
+}
+
+static void GLimp_SetHint( const char* name, const char* value )
+{
+	if ( !SDL_SetHint( name, value ) )
+	{
+		logger.Warn( "SDL_SetHint( %s, %s ) failed: %s", name, value, SDL_GetError() );
+	}
+}
+
 /*
 ===============
 GLimp_StartDriverAndSetMode
 ===============
 */
 static rserr_t GLimp_StartDriverAndSetMode( int mode, bool fullscreen, bool bordered )
 {
-#if !defined(_WIN32) && !defined(__APPLE__)
+	// See the SDL wiki page for details: https://wiki.libsdl.org/SDL3/SDL_SetAppMetadataProperty
+	GLimp_SetAppMetadataProperty( SDL_PROP_APP_METADATA_NAME_STRING, PRODUCT_NAME );
```

Comment:
There's a place in ClientApplication where the name is also set (with SDL_HINT_APP_ID) that could be cleaned up now.

Reply:
Thanks, now cleaned-up.